cdap-ui/app/cdap/components/PaidBanner/PaidBanner.scss

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+@import "../../styles/variables.scss";
+
+.paid-banner {
+  width: 30px;
+  display: inline-block;
+  position: relative;
+  color: white;
+  background-color: $green-01;
+  font-size: 13px;
+  display: block;
+  text-align: center;
+}
+.chevron {
+  position: absolute;
+  right: -8px;
+  top: 0;
+  display: block;
+  height: 16px;
+}
+.chevron:before {
+  position: absolute;
+  display: block;
+  content: "";
+  border: 9px solid $green-01;
+}
+
+.chevron:before {
+  right: 0;
+  border-right-color: white;
+}
+
+.paid-banner.large {
+  width: 160px;
+  text-align: center;
+
+  & > .chevron:before {
+    border-right-color: #e8e8e8;
```

Comment:
Try using colors from `colors.scss`, If this color is not present, check with Lea to see if a similar color that's available will work. If not, add the color into that file and then use it (check with Ajai or Edwin about this)

Reply:
This color is meant to match the background color of the card which is defined in MarketPlaceEntity.scss: 
```css
 .card-body {
      width: 100%;
      height: auto;
      z-index: 2;
      background: #e8e8e8;
      text-align: left;
       box-shadow: 2px 2px 7px;
}
```

Reply:
Done, added this color as `$grey-09`